apps/aecore/lib/aecore/txs/pool/worker.ex

```diff
@@ -67,4 +116,98 @@ defmodule Aecore.Txs.Pool.Worker do
     {:reply, tx_pool, %{}}
   end
 
+  def get_txs_for_address_with_proof(user_txs) do
```

Comment:
Proper name would be `add proof to txs`. This function doesn't *get* any txs. It is given txs and adds proof to them.